tests/Queries.elm

```diff
@@ -167,28 +167,23 @@ testIndex output =
 testChildren : Single -> Test
 testChildren output =
     describe "Query.children"
-        [ describe "on .container" <|
-            let
-                container =
-                    output
-                        |> Query.find [ class "container" ]
-            in
-                [ test "returns all the children" <|
-                    \() ->
-                        container
-                            |> Query.children []
-                            |> Query.count (Expect.equal 4)
-                , test "returns a selected child" <|
-                    \() ->
-                        container
-                            |> Query.children [ class "funky" ]
-                            |> Query.count (Expect.equal 2)
-                , test "doesn't see the nested div" <|
-                    \() ->
-                        container
-                            |> Query.children [ tag "div" ]
-                            |> Query.count (Expect.equal 0)
-                ]
+        [ describe "on the root" <|
+            [ test "sees the root has one child" <|
+                \() ->
+                    output
+                        |> Query.children []
+                        |> Query.count (Expect.equal 1)
+            , test "sees it's a <header id='heading'>" <|
+                \() ->
+                    output
+                        |> Query.children []
+                        |> Query.each (Query.has [ tag "header", id "heading" ])
+            , test "doesn't see the nested div" <|
+                \() ->
+                    output
+                        |> Query.children [ tag "div" ]
+                        |> Query.count (Expect.equal 1)
+            ]
```

Comment:
Note: @brookeangel has this fixed on a branch; waiting on https://github.com/eeue56/elm-html-query/pull/3 to get merged to implement here - at that point this revert can be dropped 😄